atom/browser/api/atom_api_app.cc

```diff
@@ -864,30 +866,43 @@ std::string App::GetLocale() {
   return g_browser_process->GetApplicationLocale();
 }
 
-bool App::MakeSingleInstance(
-    const ProcessSingleton::NotificationCallback& callback) {
+void App::OnSecondInstance(const base::CommandLine::StringVector& cmd,
+                           const base::FilePath& cwd) {
+  Emit("second-instance", cmd, cwd);
+}
+
+bool App::HasSingleInstanceLock() const {
```

Comment:
not to beat a dead horse, but what is the use case for this method being public API? :smiley: 

I'm not completely opposed to this; more like I need help understanding who would use this

Reply:
Example: A helper module for handling opening files in an Electron app, will need to know whether to expect `second-instance` events or whether to handle multiple instances 👍 

Basically helpful for third party modules that devs might use, not first party code inside apps.